build-support/scripts/devtools.sh

```diff
@@ -188,8 +188,14 @@ function install_versioned_tool {
     fi
 
     if command -v "${command}" &>/dev/null ; then
```

Comment:
I'm splitting up the `got` command into three parts. First getting the line, then getting the module name.

I've also changed the grep from `\b` to `\s` because it was incorrectly matching on `mod` for:

```
> go version -m /Users/lkysow/go/bin/mog | grep '\bmod\b'                                                                                                                                                                                                                                                                                                  ⎈ eks-1  
        mod     github.com/hashicorp/mog        v0.3.0  h1:wQkGfQAreeVrzyBiqX6nS8ZD9mtAY/MCbkclSCJJMXI=
        dep     golang.org/x/mod        v0.6.0-dev.0.20220106191415-9b9b3d81d5e3        h1:kQgndtyPBW/JIYERgdxfwMYh3AVStj88WQTlNDi2a+o=

```